pages/agent/v3/aws.md.erb

```diff
@@ -16,47 +16,68 @@ You can use an Elastic CI Stack deployment to test Linux or Windows projects,
 parallelize large test suites, run Docker containers or docker-compose
 integration tests, or perform any AWS ops related tasks.
 
-[Read the documentation][github] on GitHub, and launch it using the
+[Read the documentation][elastic-ci-github] on GitHub, and launch it using the
```

Comment:
I think as of https://github.com/buildkite/docs/pull/1077 we're recommending the Docs as the primary source.